src/pages/settings/AboutPage/ShareLogList/BaseShareLogList.tsx

```diff
@@ -91,7 +91,13 @@ function BaseShareLogList({onAttachLogToReport}: BaseShareLogListProps) {
                 <>
                     <HeaderWithBackButton
                         title={translate('initialSettingsPage.debugConsole.shareLog')}
-                        onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_CONSOLE)}
+                        onBackButtonPress={() => {
```

Comment:
This logic doesn't work correctly in some cases

https://github.com/Expensify/App/assets/9059945/84cbf582-0ff4-40f2-a356-4a9898e7950f

But I was wondering if we need the `isViaTestToolsModal` prop at all. Is its only purpose is to conditionally navigate to a different page onBackButtonPress?

Reply:
@ShridharGoel, friendly bump on this comment